website/docs/modules/train_gcnv.md

```diff
@@ -5,37 +5,132 @@ sidebar_position: 3
 slug: gcnv
 ---
 
-Trains a gCNV model for use in GatherBatchEvidence. 
-The WDL can be found at /wdl/TrainGCNV.wdl. 
-
-Both the cohort and single-sample modes use the 
-GATK gCNV depth calling pipeline, which requires a 
-trained model as input. The samples used for training 
-should be technically homogeneous and similar to the 
-samples to be processed (i.e. same sample type, 
-library prep protocol, sequencer, sequencing center, etc.). 
-The samples to be processed may comprise all or a subset 
-of the training set. For small, relatively homogenous cohorts, 
-a single gCNV model is usually sufficient. If a cohort 
-contains multiple data sources, we recommend training a separate 
-model for each batch or group of batches with similar dosage 
-score (WGD). The model may be trained on all or a subset of 
-the samples to which it will be applied; a reasonable default 
-is 100 randomly-selected samples from the batch (the random 
-selection can be done as part of the workflow by specifying 
-a number of samples to the n_samples_subsample input 
-parameter in /wdl/TrainGCNV.wdl).
-
-### Prerequisites
-
-- GatherSampleEvidence
-- (Recommended) EvidenceQC
-
-### Inputs
-
-- Read count files (GatherSampleEvidence)
-
-### Outputs
-
-- Contig ploidy model tarball
-- gCNV model tarballs
\ No newline at end of file
+import { Highlight, HighlightOptionalArg } from "../../src/components/highlight.js"
+
+[GATK-gCNV](https://www.nature.com/articles/s41588-023-01449-0)
+is a method for detecting rare germline copy number variants (CNVs)
+from short-read sequencing read-depth information.
+The [TrainGCNV](https://github.com/broadinstitute/gatk-sv/blob/main/wdl/TrainGCNV.wdl)
+module trains a gCNV model for use in the [GatherBatchEvidence](./gbe) workflow. 
+The upstream and downstream dependencies of the TrainGCNV module are illustrated in the following diagram. 
+
+<br/>
+
+```mermaid
+
+stateDiagram
+  direction LR
+  
+  classDef inModules stroke-width:0px,fill:#00509d,color:#caf0f8
+  classDef thisModule font-weight:bold,stroke-width:0px,fill:#ff9900,color:white
+  classDef outModules stroke-width:0px,fill:#caf0f8,color:#00509d
+
+  gse: GatherSampleEvidence
+  eqc: EvidenceQC
```

Comment:
I was aiming for workflow dependency, loosely reflecting and replacing the `Prerequisites` list we have. I am happy to change it to visualize the recommended execution order if that makes more sense.

Reply:
Just chiming in that if you decide you want to represent execution order, I think we can just use the overall pipeline diagram for that and additional workflow-specific diagrams aren't necessary. If you want to show dependencies, that would add new information, but I do think it could be confusing when those dependencies don't match the execution order...

Reply:
This is replicating the `Prerequisites` section we had, and apparently, it is their dependencies. I will change it for the recommended execution order, as module dependency is more code-related than useful to the end user. We can add a link to the diagram for the overall picture, and it is informative to have these diagrams to show what is at the upstream and downstream of each module to better navigate in the pipeline.